src/main/java/nova/core/recipes/crafting/ShapedCraftingRecipe.java

```diff
@@ -261,7 +261,7 @@ public void consumeItems(CraftingGrid craftingGrid) {
 			//if (original == consumed)
 			//    continue;
 
-			if (consumed.count() == 0) {
+			if (consumed != null && consumed.count() == 0) {
```

Comment:
The implementation of [`OreItemIngredient.consumeOnCrafting(Item, CraftingGrid)`](https://github.com/NOVA-Team/NOVA-Core/blob/master/src/main/java/nova/core/recipes/crafting/OreItemIngredient.java#L77) returns `null`.